.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "mounts"]
 	path = mounts
 	url = https://github.com/usegalaxy-eu/mounts
+[submodule "files/grafana"]
+	path = files/grafana
+	url = https://github.com/usegalaxy-eu/grafana-dashboards.git
```

Comment:
I am not sure if the `.git` could lead to problems, but I am also not an expert, was just noticing, that mount doesn't have it

Reply:
Should be fine afaik, that's even what GitHub suggests

![grafik](https://github.com/user-attachments/assets/c5b89e60-f7a4-4384-9f36-3d839b66e3dd)

Reply:
ok :)
I just saw it at [git handbook](https://git-scm.com/book/en/v2/Git-Tools-Submodules) without and was not sure about this